lib/resources/r4/related_person_patch.yaml

```diff
@@ -128,7 +139,47 @@ operations:
       }
     note: >
       <ul>
-        <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+        <li>Requires a <a href="#test-relationship-id">test operation</a> to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+      </ul>
+
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` extensions of the relationship at the given <code>{index}</code>.
```

Comment:
I think the description to just generally state `Replace the relationship extension field at the given index`